encode/hevcencode.c

```diff
@@ -2031,6 +2031,12 @@ static int init_va(void)
     for (i = 0; i < VAConfigAttribTypeMax; i++)
         attrib[i].type = i;
 
+    if (lowpower)
+    {
+        entryPoint = VAEntrypointEncSliceLP;
+        LCU_SIZE = 64;
```

Comment:
why not use query function to set the LCU_SIZE

Reply:
This is a sample AP.   It is simple to set 64.
We can query the size by this key VAConfigAttribEncHEVCBlockSizes, I must deal with the complicated logic for LCU_SIZE as follows.

- If the value is  nonexistent or fail to query,  I must set it to appropriate value.

- There are two values for LCU size,  I set LCU_SIZE with the min or max, or choose one value between them.
  log2_max_coding_tree_block_size_minus3
  log2_min_coding_tree_block_size_minus3